perf/perf_stress_ng.py

```diff
@@ -36,14 +36,15 @@ class Stressng(Test):
 
     def setUp(self):
         smm = SoftwareManager()
+        distro_name = distro.detect().name
         self.timeout = self.params.get('timeout', default='1')
         self.cpu_per = self.params.get("cpu_load", default='10')
         self.profile_dur = int(self.params.get("profile_duration", default=1))
-        run_type = self.params.get('type', default='upstream')
+        run_type = self.params.get('type', default='distro')
         dmesg.clear_dmesg()
 
         deps = ['gcc', 'perf']
-        if run_type == "upstream":
+        if run_type == "upstream" or 'SuSE' in distro_name:
```

Comment:
suse provide stress-ng as part of Open suse , please check this   https://download.opensuse.org/repositories/benchmark/SLE_15_SP5/

Reply:
I was able to install stress-ng package with this link. I have dropped stress-ng package related changes and updated code.